asab/library/providers/git.py

```
@@ -182,34 +198,37 @@ def credentials_cb(url, username_from_url, allowed_types):
+					pubkey_path = ""
+
+				L.debug(f"Using SSH key: {key_path}")
+
+				try:
+					return pygit2.Keypair(
+						username_from_url or self.User or "git",
+						pubkey_path,
+						key_path,
+						self.SSHPassphrase or ""
+					)
+				except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)